DogsList/DogsList/Modules/News/View/SectionHeader.swift

```diff
@@ -0,0 +1,74 @@
+//
+//  SectionHeader.swift
+//  DogsList
+//
+//  Created by Nick Krasnitskiy on 16.08.2021.
+//
+
+import UIKit
+
+class SectionHeader: UICollectionReusableView {
+    
+    static let reuserIdentifier = "SectionHeader"
+    
+    let stackView = UIStackView()
+    let title = UILabel()
+    let sortButton = UIButton()
+    
+    var sortButtonDidTapped: (() -> Void)?
+    
+    override init(frame: CGRect) {
+        super.init(frame: frame)
+        customizeElements()
+        setupConstraints()
+    }
+    
+    private func customizeElements() {
+        
+        stackView.axis = .horizontal
+        stackView.distribution = .fill
+        addSubview(stackView)
+        stackView.translatesAutoresizingMaskIntoConstraints = false
+        
+        title.textColor = #colorLiteral(red: 0.1443712413, green: 0.2595478296, blue: 0.2738721073, alpha: 1)
```

Comment:
do not use magic numbers, try to move colors  to some style structure

Reply:
Ok. Fixed.